dataactvalidator/scripts/loadFSRS.py

```diff
@@ -0,0 +1,32 @@
+from sqlalchemy import func
+
+from dataactcore.models.baseInterface import databaseSession
+from dataactcore.models.fsrs import FSRSAward
+from dataactvalidator.fsrs import retrieveAwardsBatch
+
+def loadFSRS(minId):
+    """Fetch and store a batch of Award/Sub-Award records, replacing previous
+    entries."""
+    awards = list(retrieveAwardsBatch(minId))
+    ids = [a.id for a in awards]
+    with databaseSession() as sess:
```

Comment:
I really like the simplicity of what you're doing here, and I vote for keeping it this way. Paging @jworcestBAH who can better speak to the original intent of the interfaces and make sure we're not missing anything here.